### PR TITLE
docs(IDE): add instructions to set-up PyCharm for development

### PR DESCRIPTION
**Pull request by tai271828**
Add the description of how I use PyCharm to develop both **frontend* and backend in the meantime (yes, including vue.js of *frontend*). I hope this doc will help contributors to boost your productivity.

The session I added is a bit rough. Please feel free to update this very simple paragraph.

### CODE REVIEW

**Comment by tai271828** [comment]
Hi everyone, I assigned many of you as reviewers of this pull request. That does not mean you need to be an expert of javascript or django. It's more like that I want to give all of you a heads-up that this doc update you may be interested in. Feel free to give comments or acks. Enjoy it!

File: backend/docs/SETUP_COMPOSE.md [hunk 1]
```diff
@@ -81,3 +81,80 @@ docker-compose logs --tail 100 -f web
...
+    - You need to tell PyCharm which service of docker-compose you want to launch
+      - I launched all services in command line, and then let PyCharm to take care of the following "web service" 
+        up and down.
+- Fill in necessary information required to launch backend
+- Use "Run" to launch process if you simply want to run the process
+- Use "Debug" to launch process if you want to use breakpoints to debug interactively.
+
+#### Source Entry Points
+
+- On submitting ohshown event
```
Comment (tai271828) [inline backend/docs/SETUP_COMPOSE.md:149]: @Neilxx this session may be the information you are looking for.

File: backend/docs/SETUP_COMPOSE.md [hunk 2]
```diff
@@ -81,3 +81,80 @@ docker-compose logs --tail 100 -f web
...
+    PyCharm. This is very handy!
+  - See [this reference](https://www.jetbrains.com/help/pycharm/run-debug-configuration-npm.html) if you need more detailed instructions
+- Use "Run" to launch frontend if you simply want to run the frontend
+- Use "Debug" to launch frontend if you want to use breakpoints to debug interactively.
+
+#### Source Entry Points
+
+Put your breakpoints in the functions you are interested in.
+
+- On creating ohshown event
```
Comment (tai271828) [inline backend/docs/SETUP_COMPOSE.md:113]: @Neilxx @yanghaochang104 I think you may add more information about this part :)

**Comment by EagleC0318** [comment]
Looks very cool!

**Pull request merged**